apps/smbmount.sh

```diff
@@ -89,10 +96,10 @@ done
 # Enter the SMB-user
 while true
 do
-    SMB_USER=$(whiptail --inputbox "Please enter the username of the SMB-user" "$WT_HEIGHT" "$WT_WIDTH" 3>&1 1>&2 2>&3)
-    if [[ "no" == $(ask_yes_or_no "Is this correct? $SMB_USER") ]]
+    SMB_USER=$(input_box "Please enter the username of the SMB-user" "$SUBTITLE")
```

Comment:
And imputbox function is also better...

So in other words, change were we use the old method. We don't want mixed code, remember? ;) :+1: